src/types.ts

```diff
@@ -47,6 +47,10 @@ export interface MulticallState {
       }
     }
   }
+
+  defaultListenerOptions?: {
```

Comment:
strike `default` - it's a user-setting, not a default setting

Reply:
done